named_entity_recognition/documentation/step-6-monitor-training.md

```diff
@@ -0,0 +1,25 @@
+# Monitor your training job
+
```

Comment:
This section could benefit from explicitly stating the steps that just ran. It would also help to state why Tensorboard is useful for monitoring. Perhaps guide the user's attention to specific screens/metrics/graphs.

Reply:
✓ done